### PR TITLE
Loadbot duration metrics

### PR DESCRIPTION
**Pull request by zivkovicmilos**
# Description

This PR adds support for loadbot duration metrics.
* Average Turn Around Time
* Fastest Turn Around Time
* Slowest Turn Around Time
* Total Loadbot Execution Time

as well as transaction block seal data:
* How many transactions are in each block
* How many blocks it took to seal all transactions

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [x] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Relevant docs PR [here](https://github.com/0xPolygon/polygon-sdk-docs/pull/39).

### CODE REVIEW

File: command/loadbot/execution.go [hunk 1]
```diff
@@ -29,19 +37,115 @@ type Configuration struct {
...
 }
 
 type Loadbot struct {
 	cfg     *Configuration
 	metrics *Metrics
 }
 
+// calcMaxTimeout calculates the max timeout for transactions receipts
+// based on the transaction count and tps params
+func (l *Loadbot) calcMaxTimeout(count, tps uint64) time.Duration {
```
Comment (Kourin1996) [inline command/loadbot/execution.go]: You can change this method to a function

File: command/loadbot/execution.go [hunk 2]
```diff
@@ -29,19 +37,115 @@ type Configuration struct {
...
+func (ed *ExecDuration) reportTurnAroundTime(
+	txHash web3.Hash,
+	turnAroundTime time.Duration,
+) {
+	ed.turnAroundMap.Store(txHash, turnAroundTime)
+	atomic.AddUint64(&ed.turnAroundMapSize, 1)
+}
+
+// setTotalExecTime sets the total execution time for a single loadbot run
+func (ed *ExecDuration) setTotalExecTime(totalTime time.Duration) {
```
Comment (Kourin1996) [inline command/loadbot/execution.go]: as actions reported, setTotalExecTime is not used

**Review by Kourin1996 — APPROVED** [review]
Looks good to me. It works locally. I've left small comments. Please check them. Thank you

File: command/loadbot/execution.go (hunk 1 above)
Comment (zivkovicmilos) [reply command/loadbot/execution.go]: Good catch

Fixed 👍 

28a6bd2

File: command/loadbot/execution.go (hunk 2 above)
Comment (zivkovicmilos) [reply command/loadbot/execution.go]: Fixed 👍 

28a6bd2

**Review by Kourin1996 — APPROVED** [review]
LGTM 👏

**Review by munna0908 — APPROVED** [review]
LGTM 🚀

**Review by 0xpanoramix — APPROVED** [review]
These are nice improvements :rocket:
LGTM !

**Review by ZeljkoBenovic — APPROVED** [review]
Looks good.

**Review by melekes — APPROVED** [review]

**Review by brkomir — APPROVED** [review]

**Pull request merged**